subprojects/gradle/buildchecks/src/main/kotlin/com/avito/android/plugin/build_param_check/BuildParamCheckPlugin.kt

```diff
@@ -28,13 +28,15 @@ open class BuildParamCheckPlugin : Plugin<Project> {
     override fun apply(project: Project) {
         val extension = project.extensions.create<BuildChecksExtension>(extensionName)
 
-        printBuildEnvironment(project)
-
         check(project.isRoot()) {
             "Plugin must be applied to the root project but was applied to ${project.path}"
         }
+        if (!project.pluginIsEnabled) return
```

Comment:
Added a property `avito.build-checks.enabled` to disable plugin completely. Disable all checks is not enough.

Decided to make a property, not a new extension property because it's a rare fallback.